nautobot_design_builder/design_job.py

```diff
@@ -77,10 +77,22 @@ def determine_deployment_name(cls, data):
         deployment_name_field = cls.deployment_name_field()
         if deployment_name_field is None:
             if "deployment_name" not in data:
-                raise DesignImplementationError("No instance name was provided for the deployment.")
+                raise DesignImplementationError("No Deployment name was provided for the deployment.")
             return data["deployment_name"]
         return data[deployment_name_field]
 
+    @classmethod
+    def determine_import_mode(cls, data):
+        """Determine the import mode, if specified."""
+        if not cls.is_deployment_job():
+            return None
+
+        if "import_mode" not in data:
+            return False
+            # TODO: not sure why not got the default to False from _get_vars
+            # raise DesignImplementationError("No import mode was provided for the deployment.")
+        return data["import_mode"]
+
```

Comment:
```suggestion
```

Reply:
I thought it would be clearer to warp in this function similar to what we had for deployment name...